tests/test_nn/test_modules.py

```diff
@@ -348,6 +348,18 @@ def _exp_score(self, h, t, p, power_norm) -> torch.FloatTensor:
         return -(h - t).pow(p).sum()
 
 
+class PairRETests(cases.TranslationalInteractionTests):
+    """Tests for PairRE interaction function."""
+
+    cls = pykeen.nn.modules.PairREInteraction
+
+    def _exp_score(self, h, r_h, r_t, t, p: float, power_norm: bool) -> torch.FloatTensor:
```

Comment:
@cthoyt 

shouldn't we test this for the functional form (only)?

Reply:
Correct. We have tests that the TranslationalInteraction module works properly, so any valid functional form doesn't need to be tested now.